### PR TITLE
xfstests: Add vmlinux-*-default.debug file in kdump log

### PR DESCRIPTION
**Pull request by SeroSun**
Add vmlinux-*-default.debug file (add around 100M in log, when kdump triggered) into kdump log. This action will convenience developer to start debug with kernel crash issue.

- Verification run: http://10.67.133.102/tests/831#downloads  (debug file included in kdump-xfs-473.tar.xz)

### CODE REVIEW

File: tests/xfstests/run.pm [hunk 1]
```diff
@@ -302,7 +306,7 @@ sub run {
         select_console('root-console');
         # Save kdump data to KDUMP_DIR if not set "NO_KDUMP"
         unless (get_var('NO_KDUMP')) {
-            unless (save_kdump($test, $KDUMP_DIR, "vmcore", "kernel")) {
+            unless (save_kdump($test, $KDUMP_DIR, ('vmcore', 1, 'kernel', 1, 'debug', 1))) {
```
Comment (asmorodskyi) [inline tests/xfstests/run.pm]: ahh , I was not aware that you can pass hash like this . anyway I think this is more readable way : 
```suggestion
            unless (save_kdump($test, $KDUMP_DIR, vmcore => 1, kernel => 1, debug => 1)) {
```

Comment (asmorodskyi) [reply tests/xfstests/run.pm]: and don't try to use github magic to apply changes :) I think it will not fit our flows ...

Comment (SeroSun) [reply tests/xfstests/run.pm]: Thank you for review, just modify it :)

File: tests/xfstests/run.pm [hunk 2]
```diff
@@ -302,7 +306,7 @@ sub run {
         select_console('root-console');
         # Save kdump data to KDUMP_DIR if not set "NO_KDUMP"
         unless (get_var('NO_KDUMP')) {
-            unless (save_kdump($test, $KDUMP_DIR, "vmcore", "kernel")) {
+            unless (save_kdump($test, $KDUMP_DIR, vmcore => 1, kernel => 1, debug => 1))) {
```
Comment (asmorodskyi) [inline tests/xfstests/run.pm]: you have two open brackets and three closing , left-over from previous version of code 
```suggestion
            unless (save_kdump($test, $KDUMP_DIR, vmcore => 1, kernel => 1, debug => 1)) {
```

**Pull request merged**